streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/ExternalLink.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.console.core.entity;
+
+import com.baomidou.mybatisplus.annotation.IdType;
+import com.baomidou.mybatisplus.annotation.TableField;
+import com.baomidou.mybatisplus.annotation.TableId;
+import com.baomidou.mybatisplus.annotation.TableName;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+
+import java.io.Serializable;
+import java.util.Date;
+
+@Data
+@TableName("t_external_link")
+public class ExternalLink implements Serializable {
+  @TableId(type = IdType.AUTO)
+  private Long id;
+
+  @NotBlank(message = "{required}")
+  private String name;
+
+  @NotBlank(message = "{required}")
+  private String linkUrl;
+
+  @NotBlank(message = "{required}")
+  private String imageUrl;
```

Comment:
about the badge, it is too complicated for the user to provide the url address. I think a good way is to implement the style of the badge on the front end, and the user only needs to provide the text to be displayed. what do you think?

Reply:
Better to let user maintain the badges on their own,  and we don't need to help persist the any info related to badges.

They can generate badge from third party easily, such as https://shields.io/.

Reply:
> Better to let user maintain the badges on their own, and we don't need to help persist the any info related to badges. They can generate badge from third party easily, such as https://shields.io/.

generate badge from third party I think this still has learning costs. We can build several styles and colors for users to choose, and also support user-defined badge colors. The text in the badge must be provided by the user.

Reply:
> Better to let user maintain the badges on their own, and we don't need to help persist the any info related to badges. They can generate badge from third party easily, such as https://shields.io/.

When introducing any component and service, as long as it is directly facing to the user use, it is necessary to consider the ease of use and the user's cost of use. The simpler the better

Reply:
Now we enable user to customise the badge in the similar way as  https://shields.io/.
Please help review again.